dojo/models.py

```diff
@@ -2712,20 +2733,32 @@ def save(self, dedupe_option=True, rules_option=True, product_grading_option=Tru
         self.numerical_severity = Finding.get_numerical_severity(self.severity)
 
         # Synchronize cvssv3 score using cvssv3 vector
+
         if self.cvssv3:
             try:
-
                 cvss_data = parse_cvss_data(self.cvssv3)
                 if cvss_data:
-                    self.cvssv3 = cvss_data.get("vector")
-                    self.cvssv3_score = cvss_data.get("score")
+                    self.cvssv3 = cvss_data.get("cvssv3")
+                    self.cvssv3_score = cvss_data.get("cvssv3_score")
 
             except Exception as ex:
                 logger.warning("Can't compute cvssv3 score for finding id %i. Invalid cvssv3 vector found: '%s'. Exception: %s.", self.id, self.cvssv3, ex)
                 # remove invalid cvssv3 vector for new findings, or should we just throw a ValidationError?
                 if self.pk is None:
                     self.cvssv3 = None
 
+        # behaviour for CVVS4 is slightly different. Extracting thsi into a method would lead to probabl hard to read code
```

Comment:
nit: typos in comment
```suggestion
        # behaviour for CVVS4 is slightly different. Extracting this into a method would lead to probably hard to read code
```